pages/privacy-policy/index.html

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Privacy Policy
-modified_date: September 20, 2018
+modified_date: June 3, 2020
```

Comment:
This date format looks good, but can you make it consistent with https://gruntwork.io/cookie-policy/, which uses a leading `0` for the day in the date (i.e. "June 02, 2020" vs. "June 2, 2020").